cypress/integration/search_for_property.spec.js

```diff
@@ -2,66 +2,147 @@
 
 import 'cypress-audit/commands'
 
-describe('Search for property', () => {
+describe('Search by work order reference, postcode or address', () => {
```

Comment:
Please update the name of this spec file to include work orders, or maybe better, use a separate file for work order searches.

Reply:
I was thinking the same thing, I think I will just rename the spec file, as splitting it on a separate file for work order searches means we will need to set up another login and start the server which I believe will slow down our specs